components/productCard/ActionBar.vue

```diff
@@ -2,34 +2,28 @@
   <div class="action">
     <div
       class="add-to-cart m-cart-addition qty-selector js-qty-selector"
-      :class="{ 'has-value': qty > 0 }"
+      :class="{ 'has-value': initQty > 0 }"
```

Comment:
Will this change on interaction?

Reply:
The jQuery on coop.se will update the style and add the updated quantity. So instead of trying to have our own (duplicated) functionality for trying to identify when a quantity is updated, I just removed it.